tap_github/__init__.py

```diff
@@ -164,25 +164,32 @@ def get_bookmark(state, repo, stream_name, bookmark_key, start_date):
         return start_date
     return None
 
-def raise_for_error(resp):
+def raise_for_error(resp, source):
+
+    content_length = len(resp.content)
+    if content_length == 0:
+        # There is nothing we can do here since Github has neither sent
+        # us a 2xx response nor a response content.
+        return
+
+    error_code = resp.status_code
     try:
-        resp.raise_for_status()
-    except (requests.HTTPError, requests.ConnectionError) as error:
-        try:
-            error_code = resp.status_code
-            try:
-                response_json = resp.json()
-            except Exception:
-                response_json = {}
-
-            message = "HTTP-error-code: {}, Error: {}".format(
-                error_code, ERROR_CODE_EXCEPTION_MAPPING.get(error_code, {}).get("message", "Unknown Error") if response_json == {} else response_json)
-
-            exc = ERROR_CODE_EXCEPTION_MAPPING.get(error_code, {}).get("raise_exception", GithubException)
-            raise exc(message) from None
-
-        except (ValueError, TypeError):
-            raise GithubException(error) from None
+        response_json = resp.json()
+    except Exception:
+        response_json = {}
+
+    if error_code == 404:
+        details = ERROR_CODE_EXCEPTION_MAPPING.get(error_code).get("message")
+        if source == "teams":
+            details += ' or the organization you specified is a personal space'
```

Comment:
@harshpatel4crest - Change the message to  `" or it is a personal account organization"`